packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -394,14 +410,14 @@ const ruleset: any = {
     /// ARM RPC rules for Put patterns
     ///
 
-    // RPC Code: RPC-Put-V1-01
-    PathForPutOperation: {
-      description: "The path for 'put' operation must be under a subscription and resource group.",
+    // RPC Code: RPC-Put-V1-01, RPC-Get-V1-11
+    PathForResourceGroupOperation: {
+      description: "The path must be under a subscription and resource group.",
       message: "{{description}}",
       severity: "error",
       resolved: false,
       formats: [oas2],
-      given: "$[paths,'x-ms-paths'].*[put]^~",
+      given: "$[paths,'x-ms-paths'].*[put,get]^~",
       then: {
         function: verifyArmPath,
```

Comment:
>verifyArmPath [](http://example.com/codeflow?start=18&length=13)

lets scope this check to tracked types

Reply:
Will scope this in next sprint. Addressed other comments.